es-app/src/guis/GuiMenu.cpp

```diff
@@ -366,7 +394,7 @@ GuiMenu::GuiMenu(Window* window) : GuiComponent(window), mMenu(window, "MAIN MEN
 	});
 
 	mVersion.setFont(Font::get(FONT_SIZE_SMALL));
-	mVersion.setColor(0xC6C6C6FF);
+	mVersion.setColor(0x5E5E5EFF);
```

Comment:
Yes. I asked @joolswills and it is acceptable. The contrast is really poor on smaller displays and it is nigh impossible to read it. Hence the change.